app/views/layouts/collections.html.erb

```diff
@@ -1,13 +1,22 @@
-<% content_for :head do %>
-  <script src="https://rawgit.com/alphagov/collections/master/app/assets/javascripts/support.js"/>
-  <script src="https://rawgit.com/alphagov/collections/master/app/assets/javascripts/modules/current-location.js"/>
-  <script src="https://rawgit.com/alphagov/collections/master/app/assets/javascripts/modules/accordion-with-descriptions.js"></script>
-<% end %>
+<!DOCTYPE html>
+<html>
+  <head>
+    <title><%= yield :title %> - GOV.UK</title>
+    <%= csrf_meta_tags %>
+    <%= javascript_include_tag "application" %>
+    <%= stylesheet_link_tag 'application', media: 'all' %>
+    <script src="https://rawgit.com/alphagov/collections/master/app/assets/javascripts/support.js"/>
```

Comment:
If you _do_ want to keep the JavaScript copied in the prototype, we don't need these script includes (although I'd rather keep this includes and not have the JS if we don't need to change it)